TYPO3.Flow/Classes/TYPO3/Flow/Http/Client/CurlEngine.php

```diff
@@ -148,9 +148,10 @@ public function sendRequest(Http\Request $request)
         curl_close($curlHandle);
 
         $response = Http\Response::createFromRaw($curlResult);
-        if ($response->getStatusCode() === 100) {
+        while (substr($response->getContent(), 0, 5) === 'HTTP/' || $response->getStatusCode() === 100) {
```

Comment:
I asked that myself too, but there is really no other information in `$curlResult`. It just appends the response headers after each other. We could only apply some more checks on the format of the content to make more sure it is indeed a HTTP Response, like checking for exact line delimiters (CRLF), exact status code match (3 digits) and possible headers in `identifier: value(;value)*?` format and followed by an empty line.